ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -159,15 +164,20 @@ private void SetSetting<T>(string key, T value)
             var values = ApplicationData.Current.LocalSettings.Values;
             values[key] = value;
 #else
-            var values = ConfigurationManager.AppSettings;
-            if (values[key] == null)
+            var configFile = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+            var settings = configFile.AppSettings.Settings;
```

Comment:
Let's just be readonly, either this can be a no-op or it can just set some internal in-memory dictionary.